scipy/spatial/distance.py

```diff
@@ -1510,29 +1502,26 @@ def pdist(X, metric='euclidean', *args, **kwargs):
 
     elif isinstance(metric, string_types):
         mstr = metric.lower()
-
-        try:
-            validate, pdist_fn = _SIMPLE_PDIST[mstr]
-            X = validate(X)
+        metric_name = _METRIC_ALIAS.get(mstr, None)
+
+        if metric_name is not None:
```

Comment:
Seems like this branch duplicates the logic in lines 1484 - 1495. Maybe they can be factored out?